python/cac_tripplanner/destinations/views.py

```diff
@@ -391,3 +395,82 @@ def get(self, request, *args, **kwargs):
 
         response = {'destinations': destinations, 'events': events}
         return HttpResponse(json.dumps(response), 'application/json')
+
+
+def return_400(message, error):
+    # Helper to return JSON error messages in a consistent format
+    error = json.dumps({
+        'msg': message,
+        'error': error
+    })
+    return HttpResponse(error, 'application/json', status=400)
```

Comment:
It's a good suggestion to update usage for the new-ish Django response type, but will require a larger refactor than what this PR touches, as we're not using `JsonResponse` otherwise, and should be consistent.